cmd/proxy/Dockerfile

```diff
@@ -8,7 +8,7 @@ WORKDIR $GOPATH/src/github.com/gomods/athens
 
 COPY . .
 
-ARG VERSION="Not Specified"
+ARG VERSION="unset"
 
 RUN DATE="$(date -u +%Y-%m-%d-%H:%M:%S-%Z)" && GO111MODULE=on CGO_ENABLED=0 go build -mod=vendor -ldflags "-X github.com/gomods/athens/pkg/build.version=$VERSION -X github.com/gomods/athens/pkg/build.buildDate=$DATE" -o /bin/athens-proxy ./cmd/proxy
```

Comment:
can we remove the first `&&` here, it's unnecessary or worse doesn't register DATE for the next command?

Reply:
I think it is available in the next command, here's my expert test:

```console 
~$ FARTS=on && echo $FARTS
on
```

But yes, I think I can remove it. I'll wait for a green build then tweak.

Reply:
@carolynvs LOL, thank you!